src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const countriesList = Array.from(document.querySelectorAll('span.population'));
+
+const totalPopulation = countriesList.reduce(
+  (acc, span) => acc + parseInt(span.textContent.split(',').join('')),
+  0,
+);
```

Comment:
will be more readable
```suggestion
const totalPopulation = countriesList.reduce((acc, span) => (
  acc + parseInt(span.textContent.split(',').join(''))
), 0);
```

Reply:
Prettier thinks different😅